sanity/schemas/components/grid2Images.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  name: 'grid2Images',
+  title: 'Dwa zdjęcia ze źródłem',
+  type: 'object',
+  fields: [
+    {
+      title: 'Zdjęcia',
+      name: 'images',
+      type: 'array',
+      of: [{type: 'image'}],
+    },
+    {
+      title: 'Źródło',
+      name: 'source',
+      type: 'markdown',
```

Comment:
- 'Źródło' is incomprehensible.
- This field is for URL - why type of that is markdown. Use `[URL](https://www.sanity.io/docs/url-type)` type.